modules/workflow_config/main.tf

```diff
@@ -0,0 +1,8 @@
+module "workflow_config" {
+  count     = var.deploy_workflow_config == true ? 1 : 0
+  source    = "./workflow_config"
+  namespace = var.namespace
+
+  namespace_annotations = var.namespace_annotations
+
+}
```

Comment:
We should create a secret with the access key, secret access key, region, and session token, and pass that secret name to the helm chart

Reply:
The helm chart itself creates a secret with those keys...  so, why do we need to re-create it in Terraform?